pinot-common/src/main/java/org/apache/pinot/common/request/PinotQuery.java

```diff
@@ -593,6 +604,28 @@ public void setQueryOptionsIsSet(boolean value) {
     }
   }
 
+  public boolean isExplain() {
+    return this.explain;
+  }
+
+  public void setExplain(boolean explain) {
+    this.explain = explain;
+    setExplainIsSet(true);
```

Comment:
What's the purpose of this method? If it's necessary, should it set the value from the `explain` parameter?

Reply:
This is a thrift generated method to set a flag which indicates whether a specific field is set or not.

Reply:
> Can we also add some tests under integration tests, like `OfflineClusterIntegrationTest`?
Added one integration test.